apps/pages/src/components/ListPages.vue

```diff
@@ -23,23 +23,19 @@
 </template>
 
 <script>
-import {ShowMore} from "@mswertz/emx2-styleguide";
-
 export default {
-  components: {
-    ShowMore
-  },
   props: {
-    session: Object
+    session: Object,
   },
   computed: {
     pages() {
       if (this.session && this.session.settings) {
         return Object.keys(this.session.settings)
-          .filter(key => key.startsWith("page."))
-          .map(key => key.substring(5));
+          .filter((key) => key.startsWith("page."))
+          .map((key) => key.substring(5));
       }
-    }
-  }
+      return [];
```

Comment:
explicit else would be nice

Reply:
some people say 'never use else'. I kind buy into this idea.

https://google.com/search?q=never%20use%20else